docs/openstack/create-and-connect-to-the-VM/ssh-to-the-VM.md

```diff
@@ -169,4 +181,137 @@ Once you log into the VM, you can create another user like this.
       $ chmod 600 authorized_keys
     ```
 
+## How To Enable Remote Desktop Protocol Using xrdp on Ubuntu
+
+### Log in to the server with Sudo access
+
+In order to install the `xrdp`, you need to login to the server with `sudo` access
+to it.
+
+    ssh username@your_server_ip
+
+For example:
+
+    ssh ubuntu@199.94.60.66
+
+### Installing a Desktop Environment
+
+After connecting to your server using SSH and update the list of available packages
+using the following command:
+
+    sudo apt update
+
+Next, install the `xfce` and `xfce-goodies` packages on your server:
+
+    sudo apt install xfce4 xfce4-goodies -y
+
+You will be prompted to choose a display manager, which is a program that manages
+graphical login mechanisms and user sessions. You can select any option from the
+list of available display managers, for example here we are using `gdm3`.
```

Comment:
here -> below

Reply:
I added a screenshot to show this part inside an info box.